core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -565,6 +567,8 @@ oppia.directive('conversationSkin', [
                           newStateName, _nextFocusLabel) +
                         ExplorationPlayerService.getRandomSuffix());
                     }
+                    RefresherExplorationConfirmationModalService.
```

Comment:
Before this, you'll need to check whether the exploration that's being redirected to exists and is public -- perhaps by making a call to library.ExplorationSummariesHandler in the backend. This is an asynchronous check, but it should be OK to display the modal only when the response arrives.

(The status of the exploration may have changed since it was added to the current exploration as a refresher one.)

Reply:
Done